Agasc unit test fails in ska3 prime

The failure might be an issue of the test data file format. This is the error:
```
=========================== short test summary info ============================
FAILED agasc/tests/test_obs_status.py::test_update_obs - TypeError: Cannot co...
============= 1 failed, 58 passed, 8 warnings in 61.62s (0:01:01) ==============
Traceback (most recent call last):
  File "/export/jgonzale/ska_testr/outputs/logs/Linux_2023-01-25T18-55-33_2023.1rc4-b4b8c9b_kady.cfa.harvard.edu/agasc/test_unit.py", line 2, in <module>
    testr.testr()
  File "/export/jgonzale/github-workflows/miniconda3/envs/ska3-prime/lib/python3.10/site-packages/testr/runner.py", line 67, in testr
    return test(*args, **kwargs)
  File "/export/jgonzale/github-workflows/miniconda3/envs/ska3-prime/lib/python3.10/site-packages/testr/runner.py", line 203, in test
    raise TestError('Failed')
testr.runner.TestError: Failed
```

More details:
https://icxc.cfa.harvard.edu/aspect/skare3/testr/releases/2023.1rc4-HEAD/agasc/test_unit.log

--- a/agasc/tests/test_obs_status.py
+++ b/agasc/tests/test_obs_status.py
@@ -599,9 +599,9 @@ def test_update_obs(monkeypatch, mock_open):
                 (56314, 115347520, 1, 'removed because I felt like it', '2010:110:14:57:43.442')],
                 dtype=[('obsid', '<i4'), ('agasc_id', '<i4'), ('status', '<i4'),
                        ('comments', '<U80'), ('mp_starcat_time', '<U21')]))
-            print(ref)
-            print(args[0])
-            assert np.all(args[0] == ref)
+            ref.pprint(max_lines=30)
+            args[0][ref.dtype.names].pprint(max_lines=30)
+            assert np.all(args[0][ref.dtype.names] == ref)
     mock_write.n_calls = 0
     mock_write.calls = []
 

